src/Components/Web/src/Forms/InputFile.cs

```diff
@@ -49,7 +47,6 @@ public class InputFile : ComponentBase, IInputFileJsCallbacks, IDisposable
     /// <inheritdoc/>
     protected override void OnInitialized()
     {
-        _jsUnmarshalledRuntime = JSRuntime as IJSUnmarshalledRuntime;
     }
```

Comment:
Can we now remove the whole `OnInitialized` override? Or are you keeping it to avoid churn on the public API?

Reply:
I left it like that to avoid changing the public API, but I'm definitely happy to remove it if you think that's not a concern!

Reply:
Update: I've removed this.